src/components/TheSidebar.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <button class="collapse-icon" @click="toggleSidebar">
+    <i class="fa-solid fa-bars"></i>
+  </button>
+
+  <div class="sidebar" :class="{ collapsed: collapsed }">
+    <div class="sidebarlink">
+      <router-link to="/messages" class="link"
+        ><i class="fas fa-envelope"></i><span>Messages</span></router-link
+      >
+      <router-link to="/schedules" class="link"
+        ><i class="fas fa-calendar-check"></i
+        ><span>Schedules</span></router-link
+      >
+      <router-link to="/triggers" class="link"
+        ><i class="fas fa-comment-alt"></i><span>Triggers</span></router-link
+      >
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TheSidebar',
+  data: () => {
+    return {
+      collapsed: false,
+    };
+  },
+  methods: {
+    toggleSidebar() {
+      this.collapsed = !this.collapsed;
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.sidebar {
+  background-color: var(--primary);
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  transition: 0.3s ease;
+  display: flex;
+  flex-direction: column;
+}
+
+.sidebarlink {
```

Comment:
This class makes no semantic sense, it is the wrapper of sidebar links, so name it appropriately